CHANGELOG.rst

```diff
@@ -25,6 +25,5 @@ master
 - (`#83 <https://github.com/openclimatedata/openscm/pull/83>`_) Rename OpenSCMDataFrame to ScmDataFrame
 - (`#78 <https://github.com/openclimatedata/openscm/pull/78>`_) Added OpenSCMDataFrame
 - (`#44 <https://github.com/openclimatedata/openscm/pull/44>`_) Add timeframes module
-- (`#41 <https://github.com/openclimatedata/openscm/pull/41>`_) Add DICE adapter
```

Comment:
why remove that?

Reply:
my bad didn't realise we're doing two in one